kernel/kernel-api/src/main/java/io/delta/kernel/expressions/Predicate.java

```diff
@@ -101,6 +101,11 @@
  *         <li>SQL semantic: <code>expr1 IS NOT DISTINCT FROM expr2</code>
  *         <li>Since version: 3.3.0
  *       </ul>
+ *   <li>Name: <code>STARTS WITH</code>
+ *       <ul>
+ *         <li>SQL semantic: <code>expr STARTS WITH expr</code>
+ *         <li>Since version: 3.3.0
```

Comment:
Change the version to 3.4.0. 3.3.0 is already in the process of releasing.

Reply:
Done